### DISCUSSION TITLE
I seem to be missing the cumulative entities for my export meter.

### DISCUSSION THREAD

**Discussion by jerodequin** [BottlecapDave/HomeAssistant-OctopusEnergy#1040] [General]
Hello,

Thanks for making such a useful tool!

We recently got solar and an export meter but we don't seem to have all the expected entities under the export side- 

On the export meter device we have:

Current Rate Export
Yesterday's Export
Yesterday's Export Payment

and then the following which are all disabled by default:
Off Peak Export Electricity
Current Standing Charge Export Electricity
Octoplus Saving Session Baseline Export
Previous Accumulative Cost Override
Previous Rate Export Electricity

I thought there should be a cumulative export kwh and cumulative export cost entities?

Is there something I should do to get these to appear?

Thanks!

**Comment by BottlecapDave** [comment]
Hello. Are you referring to the current accumulative sensors for export supplied from a home mini? If so, then as per the docs this is not available as the OE mini doesn't provide this data.

**Comment by jerodequin** [comment]
Hello, I do have an octopus mini but thought there would be a cumulative
value for export (based on daily yesterday's export) Is all of the export
info coming from the mini then? Sorry for misunderstanding.

**Reply by BottlecapDave** [reply]
The home mini only provides import data in "real time". The only exception to this is the demand sensor which can go negative.

When you say "cumulative value", which sensor are you meaning in particular? You said you had a following which sounds like you have them

Current Rate Export
Yesterday's Export
Yesterday's Export Payment

Could you state the default entity id of the import sensor that you expect to have an export variation of?

**Comment by twynne** [comment]
Sorry to resurrect an 'old' thread, but having recently installed a home mini I was looking for the same.  Their FAQ page at link below suggests as follows:  "You'll be able to track how much you're exporting in real-time. We're working on bringing export bars to the graphs too - watch this space!"

[https://octopus.energy/octopus-home-mini-faq/](url). (See 'Does the Home Mini support export?')

So is it the case that this information is made available to Octopus and displayed in their app, but is not available vie the API?  Or has something changed since the HA integration was developed?

**Reply by BottlecapDave** [reply]
I'm not personally on an export tariff so can't test that functionality. Last time it was looked at, the demand sensor goes negative when you're exporting, but no consumption data was available via the export meter. This might be what they are referring to in the faq. Can you see the live consumption data for your export meters in the OE app from the mini?

**Reply by twynne** [reply]
I can't at the moment, but I assumed that was an issue specific to me (I've raised with Octopus and await feedback).  I see the 'Your data's on its way' holding message but there's a section for Outgoing Octopus.  Can't unfortunately say what the Live Use is showing as we're having zero sun today, but I believe I've also seen that go negative in sunny periods.
